app/views/admin/worldwide_offices/_form.html.erb

```diff
@@ -1,58 +1,152 @@
+  } %>
+
+  <%
+    items = ContactType.all.map do |type|
+      {
+        value: type.id,
+        text: type.name,
+        checked: type.id == office_form.object.contact_type_id
+      }
+    end
+  %>
+
+  <%= render "govuk_publishing_components/components/radio", {
+    heading: "Contact type (required)",
+    name: "worldwide_office[contact_attributes][contact_type_id]",
+    id: "worldwide_office_contact_contact_type",
+    heading_size: "l",
+    error_items: errors_for(office_form.object.contact.errors, :contact_type),
+    items: items
+  } %>
+
+  <%= render "components/govspeak-editor", {
+    label: {
+      text: "Access and opening times (required)",
+      heading_size: "l",
+    },
+    name: "worldwide_office[access_and_opening_times]",
+    id: "worldwide_office_access_and_opening_times",
+    value: office_form.object.access_and_opening_times,
+  } %>
+
+  <%= render "govuk_publishing_components/components/textarea", {
+    label: {
+      text: "Comments",
+      heading_size: "l",
+    },
+    name: "worldwide_office[contact_attributes][comments]",
+    id: "worldwide_office_contact_attributes_comments",
+    value: office_form.object.comments,
+    hint: "Optional instructions for this contact"
+  } %>
+
+  <%= render "govuk_publishing_components/components/radio", {
+    heading: "Homepage feature",
+    name: "worldwide_office[show_on_home_page]",
+    inline: true,
+    heading_level: 2,
+    heading_size: "l",
+    items: [
+      {
+        value: 1,
+        text: "Yes",
+        id: "worldwide_office_show_on_home_page_1",
+        checked: worldwide_organisation.office_shown_on_home_page?(office_form.object)
+      },
+      {
+        value: 0,
+        text: "No",
+        id: "worldwide_office_show_on_home_page_0",
+        checked: !worldwide_organisation.office_shown_on_home_page?(office_form.object)
+      }
+    ]
+  } %>
+
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Email"
+    },
+    name: "worldwide_office[contact_attributes][email]",
+    id: "worldwide_office_contact_attributes_email",
+    value: office_form.object.email,
+    heading_level: 2,
+    heading_size: "l"
+  } %>
+
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Contact form URL"
+    },
+    name: "worldwide_office[contact_attributes][contact_form_url]",
+    id: "worldwide_office_contact_attributes_contact_form_url",
+    value: office_form.object.contact_form_url,
+    heading_size: "l"
+  } %>
+
+  <div class="govuk-!-margin-bottom-8">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Services",
+      font_size: "l",
+      margin_bottom: 2
+    } %>
+
+    <% # without this hidden field, once a service has been selected and saved, all services cannot be de-selected.
+       # http://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-check_box-label-Gotcha %>
```

Comment:
Are these lines needs to be removed? looks commented

Reply:
Yes, I've left these comments since they were in the legacy field and seem to provide some useful information for future reference.

Reply:
ok